lib/backends/request.js

```diff
@@ -171,7 +171,7 @@ class Request {
       method: options.method,
       uri,
       body: options.body,
-      json: true,
+      json: !(options.json === false),
```

Comment:
Do you want any falsy value for `options.json` to disable json (e.g., `undefined`), or only `false`?  If it's the former, you can make this change:

```js
const json = 'json' in options ? Boolean(options.json) : true
```

I'm good with either approach.

Reply:
it should default to true,  so undefined in this case would also be true.


If you want me to change it to what you suggested let me know

Reply:
In the code i posted `{ json: undefined }` results in setting json to `false`.  `{ }` (i.e., no `.json` in options) results in setting json to `true`.

Reply:
yea,  i guess i meant, if no `.json` is passed in, it should be true.  if you were passing in undefined, which i hope someone doesn't do, it should be false

Reply:
I've updated and squashed the other commits